python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -87,6 +87,10 @@ class ImageDifferenceConfig(pexConfig.Config):
         dtype=bool,
         default=True
     )
+    astrometer = pexConfig.ConfigurableField(
+        target = measAstrom.AstrometryTask,
+        doc = "astrometry task; used to match sources to reference objects, but not to fit a WCS",
+    )
```

Comment:
OK. Done in pipe_tasks, and that exposed one instance where I had failed to update an interface. obs_subaru has an instance of a class that has an astrometer subtask and an astrometry method; that will take some thought.